src/functions.js

```diff
@@ -66,26 +66,39 @@ defineFunction([
 defineFunction(["\\stackrel", "\\overset", "\\underset"], {
     numArgs: 2,
 }, function(context, args) {
-    const mathAxisArg = args[1];
+    const baseArg = args[1];
     const shiftedArg = args[0];
 
-    const xAxisOp = new ParseNode("op", {
+    let mclass = "mrel";
+    if (context.funcName !== "\\stackrel") {
+        // LaTeX applies \binrel spacing to \overset and \underset.
+        // \binrel spacing varies with (bin|rel|ord) of the atom in the argument.
+        // We'll do the same.
+        if (baseArg.type === "ordgroup") {
+            mclass = "m" + baseArg.value[0].type.replace("math", "");
+        } else {
+            mclass = "m" + baseArg.type.replace("math", "");
```

Comment:
Is `baseArg.type` / `baseArg.value[0].type` guaranteed to be one of `mathbin`/`mathrel`,`mathord`?  What if there's an `ordgroup` in an `ordgroup`, for example?...  I'm not suggesting going any deeper, but I worry that we need additional checking here.

Reply:
> What if there's an `ordgroup` in an `ordgroup`?

Then the result will have a CSS class `mordgroup`, that is, a class with no style rules attached. Clearly that is not ideal, but I didn't really want to do a recursive search.

In the current KaTeX release, `\overset` and `\underset` both assign `\mathop` spacing regardless of the argument type. Would it be okay if I searched for an a (bin|rel|ord) type in the top two levels, use it if found, but use `\mathop` for a default if the `ordgroup`s go deeper?

Reply:
@edemaine I just made a commit that tries to resolve nested `ordgroups`. Thank you for the nudge. What I had before was sloppy.

Mind you, what I have now does seem particularly clean, but nothing better came to mind.